glowing_waffles/notebooks/ObserveACP.ipynb

```diff
@@ -0,0 +1,300 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 34,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class observe:\n",
```

Comment:
I think the reviewNB site made my comments mostly unclear 😮 

Can you change `class observe` to `class ObserveACP`?